cip.go

```diff
@@ -57,6 +63,11 @@ func main() {
 		"do not pass '--account=...' to all gcloud calls (default: false)")
 	flag.Parse()
 
+	if *parseOnlyPtr {
+		reg.ParseManifestFromFile(*manifestPtr)
+		os.Exit(0)
```

Comment:
Does ParseManifestFromFile call Exit internally?  Otherwise this always succeeds...

Reply:
Yes, it dies with an error if the parse fails. But I can change it to return an error and make the error handling more explicit.